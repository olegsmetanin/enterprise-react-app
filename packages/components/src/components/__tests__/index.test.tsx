/* tslint:disable:no-unused-variable */
import * as React from 'react'
/* tslint:disable:no-unused-variable */
import * as renderer from 'react-test-renderer'

interface ButtonProps { };

export class Button extends React.Component<ButtonProps, void>{
  render() {
    return (
      <button
        className="btn btn-default"
        type="submit"
      >
        {this.props.children}
      </button>
    );
  }
}

it('renders correctly', () => {
    const tree = renderer.create(
        <Button>
            qwe
        </Button>
    ).toJSON()
    expect(tree).toMatchSnapshot()
})