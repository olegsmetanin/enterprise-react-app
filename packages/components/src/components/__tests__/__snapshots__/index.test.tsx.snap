exports[`test renders correctly 1`] = `
<button
  className="btn btn-default"
  type="submit">
  qwe
</button>
`;
